en/sql/datatype.rst

```diff
@@ -1167,7 +1167,7 @@ VARCHAR(n)/CHAR VARYING(n)
 
 Variable-length character strings are represented as **VARCHAR** (*n*), where *n* represents the number of characters. If *n* is not specified, the value is specified as 1,073,741,823, the maximum length.
 
-When the length of a character string exceeds *n*, they are truncated. When character string which is shorter than *n* is stored, whitespace characters are used to fill up the trailing space for **VARCHAR** (*n*), the length of string used are stored. 
+When the length of a character string exceeds *n*, they may be truncated on the insert/update operation if the allow_truncated_string   configuration value is "yes" or error occurs if not.  When character string which is shorter than *n* is stored, for **VARCHAR** (*n*), the length of string used are stored without any trailing spaces.
```

Comment:
the length of string used are stored without any trailing spaces.
==> the length of string used are stored without ignoring any trailing spaces.

Reply:
it means the trailing space is not added in VARCHAR-type, so modification might not be needed.

Reply:
ok.